internal/parser/parser.go

```diff
@@ -917,7 +917,13 @@ func (p *parser) parseStringExpression() *ast.StringExpression {
 			}
 		default:
 			// TODO bad expression
```

Comment:
drop the todo?

Reply:
I left the TODO because we have different points in which we should use a hypothetic  `BadExpression` node, and this could be one of them.